src/styles/package.json

```diff
@@ -12,7 +12,9 @@
     "build": "npm run clean && tsc"
   },
   "dependencies": {
-    "@emotion/core": "^10.0.16"
+    "@emotion/core": "^10.0.16",
+    "@types/facepaint": "^1.2.1",
```

Comment:
Small thing: if this is just types, it should probably be in `devDependencies`.

Reply:
ow, cool, i will do that